src/main/java/de/zalando/aruha/nakadi/service/StrategiesRegistry.java

```diff
@@ -9,20 +9,13 @@
 public class StrategiesRegistry {
 
     public static final PartitionStrategyDescriptor HASH_PARTITION_STRATEGY =
-            new PartitionStrategyDescriptor("hash", "This strategy will use the event " +
```

Comment:
I would agree if the registry was dynamic. 

Removing it from the API documentation would require users interested in using our API to get a token, learn about our environments or even deploy a locally, assemble the proper curl with headers and authentication just to read an excerpt of immutable text.

I would not discard the possibility of this changing in the future, but I would prefer adding complexity when it's required and not before hand.